broker/thinBroker.go

```diff
@@ -2494,12 +2472,12 @@ func (tb *ThinBroker) updateLastSuccessParameters(success bool, time string, sid
 func (tb *ThinBroker) LDUpdateEntityAttributes(w rest.ResponseWriter, r *rest.Request) {
 	var context []interface{}
```

Comment:
[Major] Check for remote broker attribute update is missing. 
Remote broker sends error, but current broker receives no error in the following scenario
updating attributes of an entity from remote broker, provided, an entity cannot be created from remote broker, it should give 409 conflict

Reply:
added support in code